charts/ccm-hetzner/values.yaml

```diff
@@ -2,6 +2,14 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
+# hccm program command line arguments.
+# The "--allocate-node-cidrs" + "--cluster-cidr" + "--leader-elect" arguments are managed by the chart and should *not* be set directly here.
+args:
+  cloud-provider: hcloud
+  allow-untagged-cloud: ""
+  configure-cloud-routes: "false"
```

Comment:
@qveensi your PR adds `--configure-cloud-routes=false` by default. Was this intentional? If "yes", why was this done?